Documentation/operations/metrics.rst

```diff
@@ -196,6 +196,7 @@ Name                                         Labels
 ``endpoint_regenerations_total``             ``outcome``                                        Count of all endpoint regenerations that have completed
 ``endpoint_regeneration_time_stats_seconds`` ``scope``                                          Endpoint regeneration time stats
 ``endpoint_state``                           ``state``                                          Count of all endpoints
+`unreachable_health_endpoints``                                                                 Number of health endpoints that cannot be reached
```

Comment:
Looks like there's a ` character missing at the start here.

Reply:
@joestringer @pchaigno thanks for the feedback . And yes this looks appealing that they should lie under a new category . I will fix this and update the pr